src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -113,6 +112,15 @@ class DraftAdjudicationService(
     )
     draftAdjudicationRepository.delete(draftAdjudication)
   }
+
+  fun getCurrentUsersInProgressDraftAdjudications(): Set<DraftAdjudicationDto> {
+    val username = authenticationFacade.currentUsername ?: return emptySet()
+
+    return draftAdjudicationRepository.findByCreatedByUserId(username)
+      .sortedBy { it.incidentDetails.dateTimeOfIncident }
+      .map { it.toDto() }
+      .toSet()
```

Comment:
Are we guaranteed ordering with set - I think it'll be okay - looking at the docs I think set retains the iterator order and so shouldn't come out differently at a later date.

Reply:
Yeah, it's fine and there is a test that covers it. But you're right in that it's a bit confusing to use a set when we're saying this data is ordered. I'll change it in the next PR. Thanks, Richard.